py/torch_tensorrt/dynamo/backend/__init__.py

```diff
@@ -57,6 +59,13 @@ def compile(
         + "torch_executed_ops, pass_through_build_failures}"
     )
 
+    if "use_experimental_fx_rt" in kwargs:
+        logger.info(
```

Comment:
Dont think you need to log this. If anything just print out what runtime is being used